pkgs/development/python-modules/pyotgw/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyserial-asyncio
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pyotgw";
+  version = "unstable-2021-03-25";
```

Comment:
Why not 1.1b1?

Reply:
The tests were added after 1.1b1.

Reply:
Request for new release: https://github.com/mvn23/pyotgw/issues/33